docs/how-tos/mutually-exclusive-experiments.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 7
+---
+
+# How to run mutually exclusive experiments in Eppo
+
+Research from Microsoft has shown that in practice [interaction effects are vanishingly rare](https://www.microsoft.com/en-us/research/group/experimentation-platform-exp/articles/a-b-interactions-a-call-to-relax/). Given this, we recommend only using mutual exclusion when the overlap of two new treatments critically degrades the user experience. For instance, a design team might want to run a homepage redesign test while a marketing team might want to experiment on the headline messaging on the same page. If the new layout removes the messaging altogether or makes it unreadable, these experiments should clearly not overlap.
+
+If you do find yourself running an experiment where there would be a true interaction effect, we recommend setting up nested feature flags with [targeting rules](/feature-flags/targeting). Using the same homepage example where each experiment has a control and one treatment, you would create 3 feature flags total:
+
+- one flag called `exclusion_group` will randomize the users into being in either the `copy_test` or `layout_out` test
+- one flag called `copy_test` will randomize users into the variations set up to test the new messaging against the control
+- one flag called `layout_test` will randomize users into the variations setup to test the new layout against control
+
+The feature flag naming convention could look like this:
+
+|  | Feature Flag name | Variation Name | Variation Name |
+| --- | --- | --- | --- |
+| Feature Flag 1 | mutual_exclusion_group | copy_test | layout_test |
+| Feature Flag 2 | copy_test | copy_control | copy_treatment |
+| Feature Flag 3 | layout_test | layout_control | layout_treatment |
+
+:::note
+
+Follow the quickstart [here](/feature-flag-quickstart/) for additional guidance on how to set up an Eppo Feature Flag in the app.
+
+:::
+
+## Eppo App Setup
+
+For your first flag, your setup should look like this:
+
+![Screenshot 2023-11-06 at 3.51.28 PM.png](/img/how-tos/mutually-exclusive-experiments/exclusion-group-flag-setup.png)
+
```

Comment:
Can we add a sentence (and maybe screenshot) showing how to randomize the `exclusion group`  flag itself? Just to make it clear that there is another layer of randomization here.

Reply:
The purpose of the first flag to randomize selection into the next 2 flags is mentioned in the untitled intro paragraph. I'll go ahead and leave out for now, but I'm happy to update/reword if we get feedback.